src/scripts/modules/media/body/content-common.less

```diff
@@ -377,7 +377,11 @@ table {
 
   // Cells
   thead,
-  tbody,
+  tbody {
```

Comment:
Should this apply to all books or only to baked books?

Reply:
I would say all books. If it has the `data-align="right"` I just want to make sure that it's actually aligning right everywhere @helenemccarron